twindle-web/team_details/persons.js

```diff
@@ -19,5 +19,23 @@ export const data = {
         country: "India",
       },
     },
-  ]
-}
\ No newline at end of file
+    {
+      id: uuidv4(),
+      name: "Satyaki Bose",
+      img:
+        "https://avatars1.githubusercontent.com/u/25426670?s=400&u=7cc72ca148ae88bf19a10b8f36b21806504ffe34&v=4",
```

Comment:
**Quick Tip:**

Instead of using such a long url, you can simply use this url:

```html
https://github.com/satyaki07.png?size=400
```

Based on tip #492

Reply:
Just like @arthurbuhl said you can use the shorter version which gives the same result @satyaki07 Do try this in your url to see the effect.